soccer/vision/ball/BallBounce.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <vector>
+
+#include <Geometry2d/Point.hpp>
+#include <Configuration.hpp>
+
+#include "KalmanBall.hpp"
+#include "vision/robot/WorldRobot.hpp"
+
+class BallBounce {
+public:
```

Comment:
=P you can probs go ahead and delete the constructor or destructor (or just turn this into a namespace if you want)

Reply:
I think I'm going to keep it a class just to hid some of the helper functions

Reply:
Well if you just want to hide it namespaces (anonymous if you really want to hide it from any other file) are great.